Code/GraphMol/FileParsers/test_data/atropisomers/AtropManyChiralsEnhanced.sdf.expected2.sdf

```diff
@@ -49,7 +49,7 @@ M  V30 19 1 17 19 CFG=1
 M  V30 20 2 6 20
 M  V30 END BOND
 M  V30 BEGIN COLLECTION
-M  V30 MDLV30/STEREL1 ATOMS=(3 10 14 7)
```

Comment:
original of this file is correct - there are 3 entries for the first stereo group.  the third is for the atropisomer.
M  V30 MDLV30/STEREL1 ATOMS=(3 10 14 7)
is correct.